src/index.html

```diff
@@ -69,38 +61,39 @@ <h1></h1>
       </div>
       <button class="read__more">CZYTAJ WIĘCEJ <i class="fa-solid fa-circle-chevron-right"></i></button>
     </div>
-
     <div class="header__arrow">
       <a href="#title">
-        <div class="header__arrow-span">
+        <div class="header__arrow-span" title="Przewiń do nowości filmowych">
           <span></span>
           <span></span>
           <span></span>
         </div>
       </a>
     </div>
-
     <div class="movie__content">
       <div class="movie__content-btn">
         <i class="fa-solid fa-circle-chevron-down"></i>
       </div>
       <div class="movie__content-img"></div>
+      <div class="movie__content-trailer">
+        <iframe class="video-frame" src="" frameborder="0" allowfullscreen></iframe>
+        <i class="fa-solid fa-square-xmark close-button"></i>
```

Comment:
Button powinien być `<button>` same stylowanie tagu `<i>` żeby wyglądał jak button nie spełnia wymogów dostępności oraz nie wspiera np takich rzeczy jak nawigowanie za pomocą `tab`

Ten "button" jest również niewidoczny dla czytników ekranu

https://a11y-101.com/development/icons-and-links